cloudify/tests/test_utils.py

```diff
@@ -65,3 +67,35 @@ def test_executable_no_override(self):
     @mock.patch.dict(os.environ, {'CFY_EXEC_TEMP': '/fake/temp'})
     def test_executable_override(self):
         self.assertEqual('/fake/temp', get_exec_tempdir())
+
+
+class TestPluginFunctions(unittest.TestCase):
```

Comment:
what about the other funcs ?

Reply:
Thought they're too simple. I'll add them too.